lib/rtcpeerconnection/safari.js

```diff
@@ -157,18 +151,26 @@ SafariRTCPeerConnection.prototype.createOffer = function createOffer(options) {
   // good enough for our application.
   if (options.offerToReceiveAudio && !this._audioTransceiver) {
     delete options.offerToReceiveAudio;
-    this._audioTransceiver = this.addTransceiver('audio');
+    try {
+      this._audioTransceiver = this.addTransceiver('audio');
+    } catch (e) {
```

Comment:
When does it throw an error? 
I see something related to recvonly offer and `addTransceiver` in safari release notes: https://webkit.org/blog/7760/release-notes-for-safari-technology-preview-34/
Something to do with this error?

Reply:
@syerrapragada When you call `createOffer` on a "closed" RTCPeerConnection.